web_programming/web_scrapping.py

```diff
@@ -0,0 +1,144 @@
+import csv
+import re
+
+import bs4
+import requests
+
+
+def scrap_info(request_url: str) -> None:
+    res = requests.get(request_url)
+
+    soup = bs4.BeautifulSoup(res.text, "lxml")
+
+    meta_info = {}
+    language = str(soup.html.get("lang"))
+    if language == "None":
+        meta_info["language"] = "Not mentioned"
+    else:
+        meta_info["language"] = language
+    charset = str(soup.meta.get("charset"))
+    if charset == "None":
+        meta_info["charset"] = "Not mentioned"
+    else:
+        meta_info["charset"] = charset
+
+    try:
+        title = soup.find("meta", property="og:title")["content"]
+        if title:
+            meta_info["title"] = title
```

Comment:
Let's use the walrus operator...
```suggestion
        if title := soup.find("meta", property="og:title")["content"]:
            meta_info["title"] = title
```
Repeat below.

Reply:
Done in the latest commit